src/RequestInspector.tsx

```diff
@@ -228,7 +228,7 @@ function ResponseDisplay({request} : { request: Request}) {
   useEffect(() => {
     let ignore = false
     if (isChromeRequest(request)) {
-      request.getContent((content, encoding) => {
+      request.getContent((content: string, encoding: string) => {
```

Comment:
I think these types should be inferred - doesn't really hurt to have them but I don't think this change is necessary